styles/youtube/catppuccin.user.css

```
@@ -1175,8 +1175,8 @@
     }
 
     /* Live chat replay */
-    .YtVideoMetadataCarouselViewModelHost {
-      background-color: @surface0;
+    .ytVideoMetadataCarouselViewModelHost {
```

I meant to explicitly say this but I tried to imply in my comment that this needs to be *in addition to*, not instead of, the existing class.

> Youtube is doing this werid thing where half of their classes start with .yT and the other half with .Yt now lol.

We need to target the classes with both the uppercase and lowercase y.
```suggestion
    .ytVideoMetadataCarouselViewModelHost, .YtVideoMetadataCarouselViewModelHost {
```